tests/ClientSpec.hs

```diff
@@ -21,6 +22,9 @@ import Servant.Client
 import Pact.Types.Runtime
 import Pact.Types.PactValue
 
+#if ! MIN_VERSION_servant_client(0,16,0)
+type ClientError = ServantError
+#endif
```

Comment:
This is the simplest way for us to get testing Chainweb with GHC 8.8. No semantics change - it's just a renaming.

It's strictly a temporary measure here and in the other test modules. Even with the recent Nix bump to GHC 8.6, we were left with the `servant-0.15` series. We know, however, that Pact works just fine with `0.16`.

If we would like the Servant up-pin, it's much easier to do now, and can be handled in a follow-up PR.